crates/moon/tests/test_cases/moon_new/mod.rs

```diff
@@ -418,14 +816,97 @@ fn test_moon_new_snapshot_lib_no_license() {
     snapbox::cmd::Command::new(moon_bin())
         .current_dir(&dir)
         .args(["new", "--lib", "hello_lib", "--no-license"])
```

Comment:
So now `--lib` has no effect. Either remove it, or deprecate it and add a warning?